src/Tizen.NUI/src/public/Layouting/GridLayout.cs

```diff
@@ -43,369 +97,291 @@ public enum Orientation
             Vertical
         }
 
-        private Orientation _GridOrientation = Orientation.Vertical;
-        const int AUTO_FIT = -1;
-        private int _columns = 1;
-        private int _rows = 1;
-        private int _totalWidth;
-        private int _totalHeight;
-        private int _requestedColumnWidth = 1;
-        private int _numberOfRequestedColumns;
-        private int _requestedRowHeight = 1;
-        private int _numberOfRequestedRows;
-        internal GridLocations _locations;
-
+        /// <summary>
+        /// [Draft] Enumeration for the alignment of the grid layout items
+        /// </summary>
+        public enum Alignment
+        {
+            /// <summary>
+            /// At the start of the container.
+            /// </summary>
+            Start,
+            /// <summary>
+            /// At the end of the container.
+            /// </summary>
+            End,
+            /// <summary>
+            /// At the center of the container
+            /// </summary>
+            Center,
+            /// <summary>
+            /// Fill the container
+            /// </summary>
+            Fill,
```

Comment:
I think we need to seperate horizontal / vertical alignment.
And I'm not sure Fill can be one of options for Alignment because it looks like sizing, not Alignment.

Fill should be applied by speicification (LayoutParamPolicy.MatchParent)

Reply:
There are `CellHorizontalAlign` and `CellVerticalAlign`. we don't need horizontal / vertical alignment here.

Alignment determines its position and size within its cell space. `Fill` can be one of options of Alignment.
`Fill` is also used in Xamarin and Android.
Xamarin : https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.layoutoptions?view=xamarin-forms
Android : https://developer.android.com/reference/android/view/Gravity

Reply:
I mean the word "Alignment" is usually used to APIs which aligns item position in NUI.

So if you want to bind sizing and positioning , it is better to change name of property as other platform( Gravity, LayoutOption).

Maybe CellOption? I'm not sure.

Reply:
`HorizontalStretch`, `VerticalStretch` are added instead of `Alignment.Fill`.